znc/content.md

```diff
@@ -25,3 +25,58 @@ The port (12345 in the example above) should match the port you used during `--m
 If you use any external module, put the .cpp, .py or .pm file to `/znc-data/modules` (you may need to create that directory).
 
 This image contains the latest released version. If you want the bleeding edge (unstable) version, it's at [zncbouncer/znc-git](https://hub.docker.com/r/zncbouncer/znc-git).
+
+## Install using [`docker-compose`](https://github.com/docker/compose)
+
+Example `docker-compose.yml` for `znc`:
+
+```yaml
+  znc:
+    image: znc:latest
+    container_name: znc
+    volumes:
+      - /your/local/znc-data:/znc-data
+    ports:
+      - 12345:12345
+    restart: unless-stopped
+```
+
+Then run with `docker-compose up -d`, and when initialized go to `http://dockerhost:12345`
+
+### Running ZNC for first time (No config)
+
+If this is your first time running ZNC (and have no configuration file), use the following `docker-compose.yml`:
+
+```yaml
+  znc:
+    image: znc:latest
```

Comment:
Since this one _won't_ be able to follow that pattern, it can take advantage of our templating (so that it's correct on the per-architecture image pages too):

```suggestion
    image: %%IMAGE%%
```